src/components/Recommend/RecommendModal.tsx

```diff
@@ -72,7 +75,7 @@ const RecommendModalBody: React.FC = () => {
         </Grid>
         <Grid key="harvard" item sm={6} xs={12}>
           <h3>{harvard.sourceName}</h3>
-          <Style.SourceIntro source={harvard.description} />
+          <Style.SourceIntro source={harvard.description} linkTarget="_blank" />
```

Comment:
We need to add `rel="noopener noreferrer"` for external links too

Reply:
adding this to paper doc todo list // have to figure out how to add this to react markdown components